resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -261,8 +261,15 @@ credentials.profile.source_profile_not_found=Profile ''{0}'' references source p
 credentials.profile.unsupported=Profile ''{0}'' is not using role-based, session-based, process-based, or basic credentials.
 credentials.profile.validation_error=Failed to switch to profile ''{0}''
 credentials.retrieving=Retrieving AWS credentials
+datagrip.auth.secrets_manager=SecretsManager Auth
+datagrip.secret_id=Secret ID:
+datagrip.secretsmanager.validation.no_password=Secret {0} does not have a field named "password"
+datagrip.secretsmanager.validation.no_secret=No SecretsManager secret specified
+datagrip.secretsmanager.validation.no_username=Secret {0} does not have a field named "username"
 datagrip.validation.invalid_credential_specified=Invalid credential profile {0} selected!
 datagrip.validation.invalid_region_specified=Invalid region {0} selected!
+datagrip.validation.no_credentials_specified=No credentials selected!
```

Comment:
We have `settings.regions.none_selected` and `settings.credentials.none_selected`, can we reuse?

Reply:
Yep I should have looked first, using those instead